### PR TITLE
2.0 form_elements components

### PR DESCRIPTION
**Pull request by protoEvangelion**

### CODE REVIEW

**Comment by alexandervalencia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/pages/docs/components/form_elements.md [hunk 1]
```diff
@@ -0,0 +1,816 @@
...
+### Basic Form Elements
+
+> Add class `focus` on the label to change the color of the label.
+
+<div class="form-group">
+	<label>
+		Default text input
+		<span class="reference-mark">
+			<svg class="lexicon-icon">
+				<use xlink:href="/images/icons/icons.svg#asterisk" />
```
Comment (alexandervalencia) [inline src/pages/docs/components/form_elements.md]: The path to the icons.svg is "/vendor/clay/icons.svg"

File: src/pages/docs/components/form_elements.md [hunk 2]
```diff
@@ -0,0 +1,816 @@
...
+```
+
+</article>
+
+
+<article id="2">
+
+### Checkboxes and Radios
+
+#### Checkboxes
```
Comment (alexandervalencia) [inline src/pages/docs/components/form_elements.md]: These `<h4>`s should be `<h5>`s make sure you're transferring the same elements from the Clay html

Comment (protoEvangelion) [inline src/pages/docs/components/form_elements.md]: I had them at h5 it just looked better as h4 but I will revert ;)

**Pull request merged**